app/views/account/bits.scala

```diff
@@ -31,11 +31,11 @@ object bits:
 
   def setting(name: Frag, body: Frag) = st.section(h2(name), body)
 
-  def radios[A](field: play.api.data.Field, options: Iterable[(A, String)], prefix: String = "ir") =
+  def radios[A](field: play.api.data.Field, options: Iterable[(A, String)], valueAliases: Option[Map[Option[String], Option[String]]] = None) =
```

Comment:
... what

Reply:
(following the other comment above)

The valueAliases is specifically to to make is so what of those 4 values is equaled to the other (3->2) for the user when he looks at the menu of pref/ui. This allows to keep the old behavior of zen toggling.

Reply:
then it would be better to avoid using this `radios` function, rather than adding this strange complexity no other caller needs.

Reply:
I see; would making a function radios_with_aliases do the job ?